apps/server/src/modules/files-storage/uc/files-storage.uc.ts

```diff
@@ -102,7 +102,7 @@ export class FilesStorageUC {
 
 			return result;
 		} catch (error) {
-			this.logger.warn(`could not find file by url: ${params.url}`, error);
+			this.logger.warn(`could not find file by url: ${params.url} ${JSON.stringify(error)}`);
```

Comment:
I dislike stringifying the error here. In theory, the logger should be responsible for how to turn objects into strings, and how to format them.

Can you try if this works:
```
this.logger.warn({
    message: 'could not find file by url',
    url: params.url,
    error: error
})
```
I think the logger should be able to accept an object, and format it correctly, but we should try to be save.

Reply:
I have problems passing it because of "Unsafe assignment of an `any` value." Struct does have the same problem.

Reply:
okay, moved it inside the method. Still not 100% satisfied...

Reply:
its because the error doesnt have a type. you can do something like this
```this.logger.warn({
				message: `could not find file by url: ${params.url}`,
				url: params.url,
				originalError: error as Error,
			});```
Still not very nice, the `as` keyword is something to be avoided wherever possible, but I think its good enough here and I wouldnt want to be stuck at this issue for too long :)

Reply:
✔️